### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by AnonymousX1025**

### CODE REVIEW

File: zerotwobot/modules/connection.py [hunk 1]
```diff
@@ -73,10 +71,9 @@ async def connection_chat(update: Update, context: ContextTypes.DEFAULT_TYPE):
 
     if conn:
         chat = await application.bot.getChat(conn)
-        chat_obj = await application.bot.getChat(conn)
-        chat_name = chat_obj.title
+        chat_name = (await application.bot.getChat(conn)).title
```
Comment (jk6521) [inline zerotwobot/modules/connection.py:74]: This is not ideal way of doing, somehow I faced errors too

File: zerotwobot/modules/connection.py [hunk 2]
```diff
@@ -118,16 +116,19 @@ async def connect_chat(update: Update, context: ContextTypes.DEFAULT_TYPE):
                 await send_message(update.effective_message, "Invalid Chat ID!")
                 return
 
-            isadmin = getstatusadmin.status in ("administrator", "creator")
-            ismember = getstatusadmin.status in ("member")
+            isadmin = getstatusadmin.status in [ChatMemberAdministrator, ChatMemberStatus.OWNER]
```
Comment (jk6521) [inline zerotwobot/modules/connection.py:119]: This should be ChatMemberStatus.ADMINISTRATOR

File: zerotwobot/modules/connection.py (hunk 1 above)
Comment (AnonymousX1025) [reply zerotwobot/modules/connection.py:74]: It works...

Comment (jk6521) [inline zerotwobot/modules/connection.py:74]: This is not ideal way of doing, somehow I faced errors too

**Review by jk6521 — CHANGES_REQUESTED** [review]

File: zerotwobot/modules/connection.py (hunk 2 above)
Comment (AnonymousX1025) [reply zerotwobot/modules/connection.py]: Looks good now.

**Pull request closed without merging**